deployment/ansible/roles/model-my-watershed.app/templates/nginx-app.conf.j2

```diff
@@ -1,3 +1,10 @@
+# Protects against scenarios where DNS records for the
+# site being proxied change.
+resolver 8.8.8.8 valid=300s;
```

Comment:
I think going with `8.8.8.8` is just the quickest way of getting one value to work across all environments. We could use a resolver provided by EC2, or something else local to the office, but then we'd have to make it a variable.